projects/main/src/app/models/cosmos/bank.application.service.ts

```diff
@@ -35,6 +36,23 @@ export class BankApplicationService {
 
     const dialogRefSimulating = this.loadingDialog.open('Simulating...');
 
+    // confirm whether amount has fee for simulation
+    const feeDenom = minimumGasPrice.denom;
+    const simulationFeeAmount = 1;
+    const tempAmountToSend = amount.find(
+      (amount) => amount.denom === minimumGasPrice.denom,
+    )?.amount;
+    const amountToSend = tempAmountToSend ? parseInt(tempAmountToSend) : 0;
+    const tempBalance = coins.find((coin) => coin.denom === minimumGasPrice.denom)?.amount;
+    const balance = tempBalance ? parseInt(tempBalance) : 0;
+    if (amountToSend + simulationFeeAmount > balance) {
+      this.snackBar.open(
+        `Insufficient fee margin for simulation!\nAmount to send: ${amountToSend}${feeDenom} + Simulation fee: ${simulationFeeAmount}${feeDenom} > Balance: ${balance}${feeDenom}`,
+      );
```

Comment:
ユーザーが任意のタイミングでスナックバーをクローズできるようにするには以下のように実装すると可能そうな気がしているので、簡易的にアラート的なユーザー通知を行うには、この書き方に統一するのがよいかもしれませんね。(松岡もこれまであまり詳細調べておらず、このような書き方できるのを、今回コメント頂いて調べて初めて把握しました。意見ご提案感謝です。)

```suggestion
      this.snackBar.open(
        `Insufficient fee margin for simulation!\nAmount to send: ${amountToSend}${feeDenom} + Simulation fee: ${simulationFeeAmount}${feeDenom} > Balance: ${balance}${feeDenom}`,
        'Close',
      );
```

エラー通知系は、一旦これに統一する形で、このプルリクでは、bank send系だけまず修正し、別のプルリクで、全体的に統一する修正を加えてあげるのが良いかなと思いました。

成功通知系は、数秒だったら消えるスナックバーの現状の実装でいいかなと思いました。